docs/docsite/rst/user_guide/playbooks_vars_facts.rst

```diff
@@ -0,0 +1,662 @@
+.. _vars_and_facts:
+
+************************************************
+Discovering variables: facts and magic variables
+************************************************
+
+With Ansible you can retrieve or discover certain variables containing information about your remote systems or about Ansible itself. Variables related to remote systems are called facts. With facts, you can use the behavior or state of one system as configuration on other systems. For example, you can use the IP address of one system as a configuration value on another system. Variables related to Ansible are called magic variables.
```

Comment:
That seems on the surface, an odd thing to do (reusing an IP address)... maybe it should say 
'as a configuration value on a replacement system' ? or some other example that doesn't end up with duplicated IP addresses...

Reply:
I was thinking of configuring a web server with the IP address of its db server. But I could definitely make that more clear! Or I'd be happy to use a different example if we can come up with a good one.